libbeat/reader/readfile/metafields_windows_test.go

```diff
@@ -70,3 +70,27 @@ func checkFields(t *testing.T, expected, actual mapstr.M) {
 
 	require.Equal(t, expected, actual)
 }
+
+func checkFieldsWithOwnerGroup(t *testing.T, expected, actual mapstr.M) {
+	t.Helper()
+
+	idxhi, err := actual.GetValue(idxhiKey)
+	require.NoError(t, err)
+	require.Equal(t, "100", idxhi)
+	err = actual.Delete(idxhiKey)
+	require.NoError(t, err)
+
+	idxlo, err := actual.GetValue(idxloKey)
+	require.NoError(t, err)
+	require.Equal(t, "200", idxlo)
+	err = actual.Delete(idxloKey)
+	require.NoError(t, err)
+
+	vol, err := actual.GetValue(volKey)
+	require.NoError(t, err)
+	require.Equal(t, "300", vol)
+	err = actual.Delete(volKey)
+	require.NoError(t, err)
+
+	require.Equal(t, expected, actual)
+}
```

Comment:
This is the same as `checkFields` above. If they ca be the same, just call `checkFields` inside `checkFieldsWithOwnerGroup`


```suggestion
func checkFieldsWithOwnerGroup(t *testing.T, expected, actual mapstr.M) {
	checkFields(t, expected, actual)
}
```

Reply:
+1

I'd also add comment explaining that on Windows we don't support adding owner and group, hence it only calls `checkFields`